yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -462,3 +462,19 @@ def test_namelist_reading_should_not_fail():
         # Test that the reading does not fail for malformed namelist.txt files
         ds = data_dir_load(ds_name)
         ds.index  # should work
+
+
+ramses_mhd_128 = "ramses_mhd_128/output_00027/info_00027.txt"
+@requires_file(ramses_mhd_128)
+def test_magnetic_field_aliasing():
+    # Test if RAMSES magnetic fields are correctly aliased to yt magnetic fields and if 
+    # derived magnetic quantities are calculated
+    ds = data_dir_load(ramses_mhd_128)
+    ad=ds.all_data()
+
+    for field in ['magnetic_field_x',
+                  'magnetic_field_magnitude',
+                  'alfven_speed',
+                  'divB']:
```

Comment:
Following the changes of #ad6d45f, this is likely to fail
 ```suggestion
                  'magnetic_field_divergence']:
```

Reply:
Well spotted. I've fixed it.